### PR TITLE
Vulnerability fixes

### PR DESCRIPTION
**Pull request by oh-vikramsingh**
**async, decode-uri-component, ejs, engine.io, http-cache-semantics, json5, minimist, node-forge, socket.io, socket.io-parser, ua-parser-js** vulnerability has been fixed
**@angular-devkit/build-angular, loader-utils, minimatch, terser, webpack** vulnerability not fixed.

### CODE REVIEW

**Review by monika-gupta-fsd — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[monika-gupta-fsd](https://github.com/scheduleonce/knowledgeowl-angular/pull/116#pullrequestreview-1479264256 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vaibhavso**
You can assign the PR to them by writing `/assign @vaibhavso` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by avOncehub — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[avOncehub](https://github.com/scheduleonce/knowledgeowl-angular/pull/116#pullrequestreview-1479266274 "Approved")*, *[monika-gupta-fsd](https://github.com/scheduleonce/knowledgeowl-angular/pull/116#pullrequestreview-1479264256 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vaibhavso**
You can assign the PR to them by writing `/assign @vaibhavso` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**
